Writing-state-migrations.md

```diff
@@ -50,25 +49,23 @@ If your PR changes the properties of an exploration or state (or other structure
+9. Fix any issues or errors from the audit job above.
+10. Get your migration PR merged.
+11. Once your PR is merged, please submit a request using this [form](https://docs.google.com/forms/d/e/1FAIpQLSfvYWscAn18ok06An1oQ54h1VmBHfCX8uuuV01kIvY9WX0-Ug/viewform) to run this migration in production. Before submitting this request, please ensure that the migration has already been tested manually on your local machine, passed code review, and been merged into develop.
 
 **Note:** These steps are for the migration where one is changing the schema of all existing states, depending on the changes your migration is going to make the steps will be less as you’ll have to change very fewer test files.
 
 If you find new test files where changes needed to be required, try updating the list.
 
 **Links to relevant PRs:**
- - Migration related to changing state schema for all possible states: [#6249](https://github.com/oppia/oppia/pull/6249)
-
- - Migration related to changing specific interaction schema: [#6177](https://github.com/oppia/oppia/pull/6177)
- - One-off job related to migration: [#6249](https://github.com/oppia/oppia/pull/6249)
+ - Deprecate math interactions' unsupported rules and update cust arg name: [#15271](https://github.com/oppia/oppia/pull/15271)
+ - Add state migration for new customization_arg catchMisspellings: [#16077](https://github.com/oppia/oppia/pull/16077). Please note that in the PR changes are done in several files and those are done according to the requirements.
 
 ## Important note:
 
-- Make sure to add a thorough test for the migration function covering each possible case. Check the PR [#11466](https://github.com/oppia/oppia/pull/11466/files) for reference.
+- Make sure to add a thorough test for the migration function covering each possible case. Check the PR [#11466](https://github.com/oppia/oppia/pull/11466/files) for reference. Please note that while you are writing tests in the `exp_domain_test.py` file you should use `swap` function to swap the CURRENT_STATE_SCHEMA_VERSION to the schema version you are writing, This way in case you have several tests the next person who will write the schema tests do not have to change `states_schema_version` of every test present. We currently do not have any specefic reference to this change, the above PR does not include this as we recently decided to go ahead with this decission. You can always take reference of `swap` function in the codebase to see the usage.
```

Comment:
```suggestion
- Make sure to add a thorough test for the migration function covering each possible case. Check the PR [#11466](https://github.com/oppia/oppia/pull/11466/files) for reference. Please note that while you are writing tests in the `exp_domain_test.py` file you should use `swap` function to swap the `CURRENT_STATE_SCHEMA_VERSION` to the schema version you are writing, This way in case you have several tests the next person who will write the schema tests do not have to change `states_schema_version` of every test present. We currently do not have any specific reference to this change, the above PR does not include this as we recently decided to go ahead with this decision. You can always take reference of `swap` function in the codebase to see the usage.
```

Reply:
Updated